apps/admin/src/pages/hackathons/hackathon-columns.tsx

```diff
@@ -0,0 +1,138 @@
+import type { ColumnDef, Row } from "@tanstack/react-table";
+import { useState } from "react";
+import { ArrowUpDown, MoreHorizontal } from "lucide-react";
+import { toast } from "sonner";
+
+import type { RouterOutput } from "@knighthacks/api";
+
+import { Button } from "~/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "~/components/ui/dropdown-menu";
+import { Sheet, SheetContent } from "~/components/ui/sheet";
+import { trpc } from "~/trpc";
+import { UpdateHackathonForm } from "./update-hackathon-form";
+
+type Hackathon = RouterOutput["hackathons"]["getAll"][number];
+
+export const columns: ColumnDef<Hackathon>[] = [
+  {
+    accessorKey: "id",
+    header: "ID",
+  },
+  {
+    accessorKey: "name",
+    header: ({ column }) => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === "asc")}
+        >
+          Hackathon Name
+          <ArrowUpDown className="ml-2 h-4 w-4" />
+        </Button>
+      );
+    },
+  },
+  {
+    accessorKey: "startDate",
+    header: ({ column }) => {
+      return (
+        <Button
+          variant="ghost"
+          onClick={() => column.toggleSorting(column.getIsSorted() === "asc")}
+        >
```

Comment:
Since the dates are in a string format, I'm curious if they'll be sorted correctly by earliest or latest start date. Have you tested this?

Reply:
I just checked they are sorted by date biggest of bros